### ISSUE TITLE
Tests fail on Windows laptop

### ISSUE THREAD

**Issue by ingorichtsmeier** [camunda/camunda-bpm-process-test-coverage#58]
Using the latest commit (https://github.com/camunda/camunda-bpm-process-test-coverage/commit/568dea90102837d451c891965d36379b9d94c46c) and import the project into Spring Tool Suite, the tests didn't run successfully.

The errors running the tests in the IDE (don't appear in maven build):
```
org.camunda.bpm.extension.process_test_coverage.rules.AggregatedCoverageTestRunStateTest$SecondNested
java.lang.AssertionError: Class coverage can only be calculated if all tests deploy the same BPMN resources. expected:<[ProcessDefinitionEntity[process-test-coverage:1:104], ProcessDefinitionEntity[super-process-test-coverage-single-branch:1:105]]> but was:<[ProcessDefinitionEntity[process-test-coverage:1:904], ProcessDefinitionEntity[super-process-test-coverage:1:905]]>
```

```
testPathAAndSuperPathA(org.camunda.bpm.extension.process_test_coverage.rules.AggregatedCoverageTestRunStateTest$SecondNested)
java.lang.NullPointerException
	at org.camunda.bpm.extension.process_test_coverage.model.MethodCoverage.addCoveredElement(MethodCoverage.java:63)
	at org.camunda.bpm.extension.process_test_coverage.model.ClassCoverage.addCoveredElement(ClassCoverage.java:31)
	at org.camunda.bpm.extension.process_test_coverage.junit.rules.DefaultCoverageTestRunState.addCoveredElement(DefaultCoverageTestRunState.java:57)
	at org.camunda.bpm.extension.process_test_coverage.listeners.FlowNodeHistoryEventHandler.handleEvent(FlowNodeHistoryEventHandler.java:52)
	at org.camunda.bpm.engine.impl.history.parser.HistoryExecutionListener.notify(HistoryExecutionListener.java:58)
...
```

`mvn clean package` results on my laptop to:
```
Results :

Failed tests:   org.camunda.bpm.extension.process_test_coverage.rules.AggregatedCoverageTestRunStateTest: No coverage information was found for target/process-test-coverage//aggregated_report/coverage/super-process-test-coverage-single-branch.html
  org.camunda.bpm.extension.process_test_coverage.rules.MultipleDeploymentsForClassTest: No coverage information was found for target/process-test-coverage//org.camunda.bpm.extension.process_test_coverage.rules.MultipleDeploymentsForClassTest/super-process-test-coverage-single-branch.html

Tests run: 46, Failures: 2, Errors: 0, Skipped: 1
```

The probels seems to be Windows-related, as the build pass on the CI (using Linux).

The maven issues depend on the regex created here: https://github.com/camunda/camunda-bpm-process-test-coverage/blob/master/test/src/test/java/org/camunda/bpm/extension/process_test_coverage/rules/AggregatedCoverageTestRunStateTest.java#L34

and evaluated here: https://github.com/camunda/camunda-bpm-process-test-coverage/blob/master/test/src/test/java/org/camunda/bpm/extension/process_test_coverage/rules/AggregatedCoverageTestRunStateTest.java#L91-L95

Maybe I will have a look into them later, but it would be nice if the authors could confirm that they use Linux computers. And if any voluteers could confirm my issues on either Mac or Windows laptops.

**Comment by ingorichtsmeier** [comment]
I found some hints.

I disabled the failing tests on my branch and run the build locally on my Windows laptop.

The percentage generation from this build contains a special character on windows: 
![grafik](https://user-images.githubusercontent.com/7870843/90245136-6015e480-de32-11ea-8062-24987d885be3.png)

With Notepad++ and special characters displayed:
![grafik](https://user-images.githubusercontent.com/7870843/90245228-8e93bf80-de32-11ea-921b-f90dd6b04400.png)

When you use the prepackaged version (0.3.2), you will get only linefeeds and some other special character:
![grafik](https://user-images.githubusercontent.com/7870843/90245387-dd415980-de32-11ea-8cc7-fd24686339de.png)

In HEX, the upper one is a0, and the lower one is c2 a0.
